tests/graphics/test_softras.py

```diff
@@ -16,19 +16,34 @@
 import torch
 import kaolin
 
+
+@pytest.mark.skipif("not torch.cuda.is_available()", reason="softras needs GPU")
 def test_softras(device='cuda:0'):
 
-    renderer = kaolin.graphics.SoftRenderer(camera_mode='look_at', device=device)
-    filename_input = 'tests/model.obj'
-    
+    renderer = kaolin.graphics.SoftRenderer(camera_mode="look_at", device=device)
+    filename_input = "tests/graphics/banana.obj"
+
     mesh = kaolin.rep.TriangleMesh.from_obj(filename_input)
-    vertices = mesh.vertices.float()
+
+    vertices = mesh.vertices
     faces = mesh.faces.long()
-    face_textures = faces.clone()
-    vertices = vertices[None, :, :].cuda()
+    face_textures = (faces).clone()
+    vertices = vertices[None, :, :].cuda()  
     faces = faces[None, :, :].cuda()
-    face_textures = face_textures[None, :, :].cuda()
+    face_textures[None, :, :].cuda()
```

Comment:
`face_textures` is unused.

Reply:
Good catch, this was copy-pasted from the old softras example